kizzanaome/nomaCakes
# #1 Create project structure using REST

### What does this PR do?
 This PR is to Iset up the nomaCakes project from scratch

### Description of Task to be completed?
 Use REST from Django and setup the project

### How should this be manually tested?
- Fetch this branch by running the command `git fetch origin ch-setup-project` in the terminal
- Check into this branch by running `git checkout ch-setup-project` in the terminal
- Activate your virtual environment by running `source nomaenv/bin/activate` in the terminal
- Create a database by typing `CREATE DATABASE <database name>;` preferably in a `psql` terminal.
- Create a .env file and use the sample env file to fill in the details required to connect to your database
- Apply the new changes to the models by running `python manage.py migrate` in the terminal
- Start the application by running `python manage.py runserver`
